draft/2023-03-29-this-week-in-rust.md

```diff
@@ -148,39 +138,82 @@ Rusty Events between 2023-03-29 - 2023-04-26 🦀
+    * [**Rust meetup #34 at Penneo**](https://www.meetup.com/copenhagen-rust-community/events/291613010/)
+* 2023-03-30 | Lille, FR | [Rust Lille](https://www.meetup.com/meetup-group-zgphbyet)
+    * [**Rust Lille #3**](https://www.meetup.com/meetup-group-zgphbyet/events/292257807/)
+* 2023-03-30 | Vienna, AT | [Rust Vienna](https://www.meetup.com/rust-vienna)
+    * [**Rust Vienna Meetup - March**](https://www.meetup.com/rust-vienna/events/292064505/)
 * 2023-04-04 | Berlin, DE | [Berline.rs](https://berline.rs)
     * [**Rust and Tell - Goodbye👋 Edition**](https://berline.rs/2023/04/04/rust-and-tell-goodbye-edition.html)
 * 2023-04-06 | Lyon, FR | [Rust Lyon](https://www.meetup.com/fr-FR/rust-lyon/)
     * [**Rust Lyon Meetup #3**](https://www.meetup.com/fr-FR/rust-lyon/events/292283973/)
 * 2023-04-19 | Zurich, CH | [Rust Zurich](https://www.meetup.com/de-DE/rust-zurich/)
     * [**sett: data encryption and transfer made easy(ier)**](https://www.meetup.com/de-DE/rust-zurich/events/292151879/)
+* 2023-04-20 | Aarhus, DK | [Rust Aarhus](https://www.meetup.com/rust-aarhus/)
+    * [**Rust Aarhus meetup #1 at Geanix**](https://www.meetup.com/rust-aarhus/events/292185072/)
+* 2023-04-20 | Munich, DE + Virtual | [Rust Munich](https://www.meetup.com/rust-munich/)
+    * [**Rust Munich 2023 / 2 - hybrid**](https://www.meetup.com/rust-munich/events/291965920/)
 * 2023-04-20 | Bern, CH | [Rust Bern](https://www.meetup.com/de-DE/rust-bern/)
     * [**First Rust Bern Meetup!**](https://www.meetup.com/de-DE/rust-bern/events/292206056/)
 
-### Asia
-* 2023-04-08 | Kyoto, JP | [Kansai Rust](https://www.meetup.com/kansai-rust/)
-    * [**Demystifying Closures**](https://www.meetup.com/kansai-rust/events/292202435/) 
-* 2023-04-08 | Beijing, China | [Rust Chinese Group](https://www.meetup.com/rust-chinese-group/)
-    * [**Rust Meetup Beijing**](https://www.meetup.com/rust-chinese-group/events/292379002/) 
-* 2023-04-12 | Kuala Lumpur, MY | [Rust Malaysia](https://rust-malaysia.github.io/meetup/); [Telegram](https://t.me/golangmalaysia)
-    * [**Rust Meetup Malaysia April 2023: How far is Dioxus from React? by Ivan Tham**](https://www.google.com/calendar/event?eid=MWI0bWJzY21qZTI2NWsyZDgzOG0xb2JkdTkgYXBkOXZtYmMyMmVnZW5tdHU1bDZjNWpiZmNAZw&ctz=America/Los_Angeles) | [Map](https://goo.gl/maps/w2ogftac6mqpBbvt5)
+### North America
+
+* 2023-03-30 | Ciudad de México, MX | [Rust MX](https://www.meetup.com/rust-mx/)
+    * [**Escribiendo Bibliotecas de sistema con Rust**](https://www.meetup.com/rust-mx/events/292168261/)
+* 2023-04-13 | New York, NY, US | [Rust NYC](https://www.meetup.com/rust-nyc/)
+    * [**Autometrics: Easily add metrics in Rust and understand them in Prometheus**](https://www.meetup.com/rust-nyc/events/292430796/)
+
+### Oceania
+
+* 2023-04-04 | Aukland, NZ | [Rust AKL](https://www.meetup.com/rust-akl/)
+    * [**Christchurch Rust meetup first meeting**](https://www.meetup.com/rust-akl/events/292262863/)
```

Comment:
This might have been copied over by accident. The title is listed as:

```suggestion
    * [**Rust AKL: Real Time Audio Editing + A New Automation Framework**](https://www.meetup.com/rust-akl/events/292262863/)
```

Reply:
Very good spot, thank you!!!